### PR TITLE
Merging fix for the Release zip.

### PR DESCRIPTION
**Pull request by slok1368**
- Original release was built in Debug mode, which requires dependencies from Visual Studio Build Tools.
- Fixed code to be built using Release mode.
- Mainly made changes to the solution configurations.

### CODE REVIEW

**Review by lochoa2 — COMMENTED** [review]
looks good

**Review by Kelbaso — APPROVED** [review]
Nice work Steven, this should fix the issues people were facing in the Beta Release.

File: src/front-end/front-end-interface/GamesInOne.csproj [hunk 1]
```diff
@@ -44,10 +44,56 @@
     <PlatformTarget>x64</PlatformTarget>
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
+    <OutputPath>..\..\..\GamesInOne-Release\x64\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <CodeAnalysisRuleSet>SecurityRules.ruleset</CodeAnalysisRuleSet>
```
Comment (asagrekin) [inline src/front-end/front-end-interface/GamesInOne.csproj:51]: Good fix!

**Review by asagrekin — APPROVED** [review]
Looks good to me. Not issues as far as I can tell. Let's merge this asap.

**Pull request merged**